locale/en/docs/es6.md

```diff
@@ -55,12 +55,26 @@ All ES6 features are split into three groups for **shipping**, **staged**, and *
 
 * [Arrow Functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
 
+* [new.target](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new.target)
+
+* [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe)
```

Comment:
Ah good catch. While I agree it shouldn't be in the ES6 section, I do think it should be mentioned (somewhere). I'll remove it from here though